### PR TITLE
Adding message to deleteKubernetesSupportedVersion

### PR DESCRIPTION
**Pull request by davidjumani**
Screenshot from 2020-11-09 10-22-30

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/locales/en.json [hunk 1]
```diff
@@ -2897,6 +2897,7 @@
 "message.kubernetes.cluster.start": "Please confirm that you want to start the cluster",
 "message.kubernetes.cluster.stop": "Please confirm that you want to stop the cluster",
 "message.kubernetes.cluster.upgrade": "Please select new kubernetes version",
+"message.kubernetes.version.delete": "Please confirm that you want to delete this kubernetes version",
```
Comment (shwstppr) [inline src/locales/en.json:2900]:
```suggestion
"message.kubernetes.version.delete": "Please confirm that you want to delete this Kubernetes version",
```

File: src/locales/en.json [hunk 2]
```diff
@@ -2897,6 +2897,7 @@
 "message.kubernetes.cluster.start": "Please confirm that you want to start the cluster",
 "message.kubernetes.cluster.stop": "Please confirm that you want to stop the cluster",
 "message.kubernetes.cluster.upgrade": "Please select new kubernetes version",
```
Comment (shwstppr) [inline src/locales/en.json:2899]:
```suggestion
"message.kubernetes.cluster.upgrade": "Please select new Kubernetes version",
```
@davidjumani can you please change this as well

**Review by shwstppr — COMMENTED** [review]
LGTM with minor message changes

**Review by Pearl1594 — APPROVED** [review]

**Review by utchoang — APPROVED** [review]
LGTM!

**Review by shwstppr — APPROVED** [review]
LGTM

**Pull request merged**